### ISSUE TITLE
Icon not working in mdc-tab

### ISSUE THREAD

**Issue by locobert** [stasson/vue-mdc-adapter#335]
I've tried the example provided on <https://github.com/stasson/vue-mdc-adapter/tree/master/components/tabs>:
```html
<mdc-tab-bar>
  <mdc-tab icon="phone">Recents</mdc-tab>
  <mdc-tab icon="favorite">Favorites</mdc-tab>
  <mdc-tab icon="personal_pin">Nearby</mdc-tab>
</mdc-tab-bar>
```
Codepen: <https://codepen.io/anon/pen/eMGbmY>

The following looks fishy to me (always returns false?): https://github.com/stasson/vue-mdc-adapter/blob/master/components/tabs/mdc-tab.vue#L40 
```JavaScript
    hasIcon () {
      if (this.icon || this.$slots.icon) {
        this.icon ? extractIconProp(this.icon) : {}
      }
      return false
    },
```

**Comment by pgbross** [comment]
Resolved in #336

**Comment by locobert** [comment]
Thanks a lot for the fast fix!
